composer.json

```diff
@@ -14,13 +14,16 @@
         "twig/twig": "^2.13"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.5"
+        "phpunit/phpunit": "^9.5",
+        "bamarni/composer-bin-plugin": "^1.4"
     },
     "autoload": {
         "psr-4": {"my127\\Workspace\\": "src/"}
     },
     "scripts": {
-        "test": "composer compile && vendor/bin/phpunit --testdox",
+        "post-install-cmd": ["@composer bin all install --ignore-platform-reqs --ansi"],
```

Comment:
Trigger installing binary dependencies after normal install or update. Currently only `humbug/box` will be installed into `vendor-bin/php-box/vendor`.